README.md

```diff
@@ -28,6 +28,8 @@ manager behavior.
 - If no `library_path` has been specified, the `BAL_LIBRARY_PATH` env
   var will be checked to see if it points to a valid library file.
 
+- Persists newly registered data in-memory.
```

Comment:
[consider][minor] Seems a bit vague to be a feature. Was the motivation behind this this actually to imply that newly registered data won't get serialized back to a `.json`?

Reply:
Exactly that. It wasn't capable of registering new data before (read-only) so it was something we explicitly added. I can elaborate a little more if you think its useful.